src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -18,7 +18,8 @@ public class JsonSerializableAddressBookTest {
     private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonSerializableAddressBookTest");
     private static final Path TYPICAL_PROFILES_FILE = TEST_DATA_FOLDER.resolve("typicalProfilesAddressBook.json");
     private static final Path INVALID_PROFILE_FILE = TEST_DATA_FOLDER.resolve("invalidProfileAddressBook.json");
-    private static final Path DUPLICATE_PROFILE_FILE = TEST_DATA_FOLDER.resolve("duplicateProfileAddressBook.json");
+    private static final Path DUPLICATE_NAME_FILE = TEST_DATA_FOLDER.resolve("duplicateNameAddressBook.json");
+    private static final Path DUPLICATE_EMAIL_FILE = TEST_DATA_FOLDER.resolve("duplicateEmailAddressBook.json");
```

Comment:
Maybe this could be renamed to `SIMILAR_NAME_FILE` and `SIMILAR_EMAIL_FILE` to maintain consistency with the renaming